audioplayer.cpp

```diff
@@ -0,0 +1,95 @@
+#include "audioplayer.h"
+
+
+void AudioPlayer::getFileNames(const string &folder, vector<string>) {
+    vector<string> names;
+    DIR *directory;
```

Comment:
In google code conventions asterisk (and ampersand) sticks to the type name, like this `DIR* directory`. This should be fixed everywhere.

I especially like that, because for me asterisk is the part of the type, not variable name :)